example/01/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/axamon/gobrain"
```

Comment:
Shouldn't it be `github.com/goml/gobrain`?

Reply:
yea fixed it